lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationRequest.cpp

```diff
@@ -26,16 +26,16 @@ AuthenticationRequestMsg::AuthenticationRequestMsg(){};
 AuthenticationRequestMsg::~AuthenticationRequestMsg(){};
 
 // Decode AuthenticationRequest Messsage
-int AuthenticationRequestMsg::DecodeAuthenticationRequestMsg(
-    AuthenticationRequestMsg* auth_request, uint8_t* buffer, uint32_t len) {
+int AuthenticationRequestMsg::DecodeAuthenticationRequestMsg(uint8_t* buffer,
+                                                             uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.